modules/core/src/shaderlib/shadow/shadow.js

```diff
@@ -118,13 +119,40 @@ function _getViewportCenterPosition({viewport, center}) {
   return new Matrix4(viewport.viewProjectionMatrix).invert().transformVector4(center);
 }
 
+function _getViewProjectionMatrix({viewport, shadowMatrix}) {
```

Comment:
No need to add underscore. If the function is not exported, it is not exposed to the user.